common/audit.cto

```diff
@@ -117,9 +117,18 @@ transaction ClauseActionFailed extends ClauseTriggerEvent {
   o String error
 }
 
+// TODO: remove test property from ClauseTriggered and ClauseTestTriggered transactions
+
 transaction ClauseTriggered extends ClauseTriggerEvent {
   o String event // JSON
   o String error optional
   o String stack optional  // Required if severity == ERROR
   o Boolean test default=false optional
 }
+
+transaction ClauseTestTriggered extends ClauseTriggerEvent {
```

Comment:
instead you could just write `transaction ClauseTestTriggered extends ClauseTriggered {}` and use the type declaration to determine the difference.

Also, why do you need to keep the test property around?

Reply:
The test property is just there temporarily for backwards compatibility. I'll check if it used elsewhere, and then remove it from the model.